### DISCUSSION TITLE
Track events

### DISCUSSION THREAD

**Discussion by ForestOfDreams** [merdok/homebridge-miot#188] [Q&A]
How can I add events in HomeKit? There is humidifier:low-water-level event in zhimi.humidifier.ca4 device, how can i use it?

**Comment by merdok** [comment]
Events at this point do not work yet, but since they are part of the spec I already added them to the devices for future investigation to get them to work.
For now you can simply use a propertyMonitor entry and monitor the humidifier:water-level property for a specific value at which you would like to get notified when the water level is low.
